addon/components/o-s-s/banner.ts

```diff
@@ -24,7 +29,7 @@ export default class OSSBanner extends Component<OSSBannerArgs> {
   }
 
   get sizeClass(): string {
-    return this.args.size === 'sm' ? 'upf-banner--size-sm' : '';
+    return SIZE_CLASSES[this.args.size ?? 'md'] ?? '';
```

Comment:
the first check is enough as SIZE_CLASSES always exists:
`return SIZE_CLASSES[this.args.size ?? 'md']`